src/bin/cut.rs

```diff
@@ -0,0 +1,525 @@
+
+
+use std::env;
+use std::fs;
+use std::io::{self, BufRead, Read, Write};
+use std::slice;
+use std::str::FromStr;
+
+// libextra is not compiling.
+use std::process;
+pub fn fail<'a>(s: &'a str, stderr: &mut io::Stderr) -> ! {
+    let mut stderr = stderr.lock();
+
+    let _ = stderr.write(b"error: ");
+    let _ = stderr.write(s.as_bytes());
+    let _ = stderr.write(b"\n");
+    let _ = stderr.flush();
+    process::exit(1);
+}
+
+
+static USAGE: &'static str = r#"usage: cut -b list [-n] [file ...]
+       cut -c list [file ...]
+       cut -f list [-s] [-d delim] [file ...]
+"#;
+
+static MAN_PAGE : &'static str = r#"NAME
+     cut -- cut out selected portions of each line of a file
+
+SYNOPSIS
+     cut -b list [-n] [file ...]
```

Comment:
I actually like the [BSD synopsis](https://www.freebsd.org/cgi/man.cgi?query=cut&apropos=0&sektion=0&manpath=SunOS+5.9&format=html) much more than the [LINUX synopsis](http://linux.die.net/man/1/cut). It takes three lines but is actually much more clear. If you still think it should be one line, then I will change it.